pydantic/typing.py

```diff
@@ -49,9 +49,16 @@ def evaluate_forwardref(type_, globalns, localns):  # type: ignore
     AnyCallable = TypingCallable[..., Any]
 
 try:
-    from typing_extensions import Literal
+    from typing import Literal as TypingLiteral
 except ImportError:
-    Literal = None  # type: ignore
+    TypingLiteral = None  # type: ignore
+
+try:
+    from typing_extensions import Literal as TypingExtensionsLiteral
+except ImportError:
+    TypingExtensionsLiteral = None  # type: ignore
+
+Literal = TypingLiteral or TypingExtensionsLiteral  # prefer TypingLiteral if possible
```

Comment:
Since we use the `Literal` type from this package in some places in the code, we prioritize the built-in, which should always be available if supported.